### DISCUSSION TITLE
SDXL load every prompt

### DISCUSSION THREAD

**Discussion by celeebi** [Comfy-Org/ComfyUI#1871] [Q&A]
hello everyone, im just getting started on using comfyui and its great so far, for a pc that has lower spec it works very great, however i have some issues with sdxl, ussualy when i used regular loader, for sd 1.5, once we load the model, it just prompt after prompt, just like A1111, however when i used  sdxl base+refiner, it loads the model everytime i queue the prompt, and ussually loading the model is taxing the time, i just wonder if there's a way to make sure the sdxl stay loaded like sd 1.5

**Comment by phineas-pta** [comment]
buy 3090/4090
